### ISSUE TITLE
Make output of doc._.quality specify if each filter was passed

### ISSUE THREAD

**Issue by HLasse** [HLasse/TextDescriptives#155]
For instance instead of returning:

```
{'n_stop_words': 435,
 'alpha_ratio': 0.79,
 'mean_word_length': 1.3,
 'doc_length': 894,
...
}
```
We could return the values as a tuple with the second value indicating whether it passed (given the current threshold)
```
{'n_stop_words': (435, True),
 'alpha_ratio': (0.79, True),
 'mean_word_length': (1.3, False),
 'doc_length': (894, True),
```

Alternatively, return a dataclass with a `.value` and a `.passed` attribute?

**Comment by KennethEnevoldsen** [comment]
Hm what about the format:
```python
doc._.passed_quality_check
# True
# great to keep it a simple boolean
doc._.quality_check
# QualityCheck(passed=True, filter_checks={"n_stop_words": True, "alpha_ratio": False ...})

# This could give rise to (competing with doc._.passed_quality_check)
doc._.quality_check.passed
```

**Comment by HLasse** [comment]
Fixed by #153
